qemu/tests/cfg/live_backup.cfg

```diff
@@ -38,3 +38,12 @@
                     granularity = 32768
                 - 131072:
                     granularity = 131072
+        - data_plane:
```

Comment:
Please rename the variant to `with_data_plane` to avoid being duplicate with other cases.